### ISSUE TITLE
Toggles a navigation handler that automatically intercepts only <a> with css selector

### ISSUE THREAD

**Issue by croban** [zyxd/svelte-store-router#7]
Could be possible to have additional navigation handler toggle that fires only if <a> tag is below an css selector?
E.g. .app-wiget a...

**Comment by zyxd** [comment]
Do I understand correctly that you need to implement the ability to pass an array (maybe an object?) with selectors to `handleHavigation`, within which the handler should work?

Something like:
```
createRouteStore({
  handleNagivation: [
    '.foo',
    '#bar',
    'form'
  ]
})
```

**Comment by croban** [comment]
Correct, basically I am using svelte app as an widget inside other websites. Widget container self is under well-known node (by id and by css class) and i would like that link handler intercept only clicks from links below that node.

**Comment by zyxd** [comment]
Could you check please v1.0.25? Parent elements now can be specified in `handleNavigation` as a comma separated CSS selector string, e.g. '.foo, #bar, form.abc'.

**Comment by croban** [comment]
Thx, ...d.ts missing boolean | string
I have add @ts-ignore so that I can continue:
```
export const route = createRouteStore({
  queryClean: true,
  fragmentClean: true,
  // @ts-ignore
  handleNavigation: "#app" 
})
```

more feedback follows

**Comment by zyxd** [comment]
Oops, my bad. Fixed in v1.0.26.

**Comment by croban** [comment]
it works for me, thx ;)
